hw3/01-characters.js

```diff
@@ -0,0 +1,43 @@
+// url for the Thrones API
+const url = 'https://thronesapi.com/api/v2/Characters';
+
+fetch(url)
+.then(response => response.json())
+.then(characters => {
+    const characterList = document.getElementById('characters');
+    
+    characters.forEach(character => {
+      const { firstName, lastName, imageUrl, title } = character;
+
+      const characterContainer = document.createElement('section');
+      characterContainer.classList.add('character-container');
+
+      const characterImage = document.createElement('img');
+      characterImage.src = imageUrl;
+      characterImage.alt = imageUrl;
```

Comment:
![Screenshot (269)](https://github.com/rutujapadgilwar/frontend_homework/assets/38127011/5361f307-659e-4cee-9fa8-ab5102e6c036)

Accessibility concerns:
1) The alternative should be descriptive. It should not be the image url.
2) On hover there is a contrast error

Reply:
Thank you for the feedback, @JoshuaWachana. I have incorporated your suggestions and made the necessary changes to the code. Now, the character elements will have improved contrast on hover, which helps address the contrast error.